src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -311,6 +328,10 @@ Status CoreWorkerMemoryStore::GetImpl(const std::vector<ObjectID> &object_ids,
         count += 1;
       } else {
         remaining_ids.insert(object_id);
+        Status status = ObjectHasOwner(object_id, abort_if_any_object_is_exception);
```

Comment:
Hmm, I think `abort_if_any_object_is_exception` and `abort_if_missing` are two different things.

Reply:
ok I've changed this